src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -102,11 +91,11 @@ class SQLPTRepresentationTest : public SQLPTRepresentation,
   }
 
   static void TearDownTestCase() {
-    delete query_wrapper_;
+    query_wrapper_.reset();
     EXPECT_TRUE(reps->Drop());
     EXPECT_TRUE(reps->Close());
     reps->RemoveDB();
-    delete reps;
+    reps.reset();
```

Comment:
@ZhdanovP do we really need reset shared pointers here?

Reply:
removed